index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="icon" href="/images/panda_face.svg" type="image/svg" />
+    <link rel="stylesheet" href="index.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-content">
+        <div class="logo-container">
+          <img src="images/panda_face.svg" alt="logo" class="logo" />
+          <a href="/"><h2>판다마켓</h2></a>
+        </div>
+        <a href="/login">
+          <button class="login-button">로그인</button>
+        </a>
```

Comment:
❗️ 수정요청
로그인 버튼을 눌렀을 때 './login'로 이동시켜주는 역할을 하니, 앵커 태그만 남겨두셔도 될 것 같습니다~
버튼 요소처럼 보여도 꼭 버튼 태그가 포함될 필요는 없습니다.
해당 태그의 역할에 따라 태그를 시맨틱하게 작성하시고 디자인은 css로 구현하시는 것이 좋습니다.

```suggestion
        <a href="/login" class="login-button">
          로그인
        </a>
```